ingestion/src/metadata/orm_profiler/profiler/core.py

```diff
@@ -91,6 +90,9 @@ def __init__(
         self.exclude_columns = exclude_columns
         self._metrics = metrics
         self._profile_date = profile_date
+        self.profile_sample_config = (
+            self.profiler_interface.profile_sample_config or None
```

Comment:
I think you can just do `self.profiler_interface.profile_sample_config`. Because in `get_profile_sample` we either return the object or None.

Reply:
resolving that